### PR TITLE
Issue #2320 - Address experiment options nits.

### PR DESCRIPTION
**Pull request by liuche**
### Pull Request checklist

- [ ] This PR includes thorough **tests** or an explanation of why it does not
- [x] This PR includes a **CHANGELOG entry** or does not need one
- [x] I have considered adding **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

File: app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt [hunk 1]
```diff
@@ -17,6 +17,7 @@ import org.mozilla.tv.firefox.telemetry.TelemetryIntegration
 import org.mozilla.tv.firefox.utils.UrlUtils
 
 data class ValidatedIntentData(val url: String, val source: Session.Source)
+private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
```
Comment (mcomella) [inline app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt]: nit: constants should be defined about classes. Also, imo classes & values should generally have newlines between them.

Comment (mcomella) [inline app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt]: nit: Maybe we should change the value to `qaActiveExperiments`? That will make it clear to QA (and pesky users!) that they should not have many expectations when using this functionality. I asked Daniela to make this change too.

Comment (mcomella) [inline app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt]: nit: The value should probably be `EXTRA_KEY_ACTIVE_EXPERIMENTS` or `KEY_ACTIVE_EXPERIMENTS`: it makes it more obvious how certain constants are to be used when it's prefixed. I asked Daniela to make this change too.

Comment (liuche) [reply app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt]: I'm reading this as "above classes" and "add a newline"

Comment (liuche) [reply app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt]: I'll update the wiki page too.

**Pull request merged**